src/ProductConstructionService/ProductConstructionService.DependencyFlow/PullRequestUpdater.cs

```diff
@@ -242,26 +242,26 @@ protected virtual Task TagSourceRepositoryGitHubContactsIfPossibleAsync(InProgre
         return Task.CompletedTask;
     }
 
-    protected async Task<PullRequestStatus?> GetPullRequestStatusAsync(InProgressPullRequest pr, bool isCodeFlow)
+    protected async Task<PullRequestStatus?> GetPullRequestStatusAsync(InProgressPullRequest pr, bool isCodeFlow, bool tryingToUpdate)
```

Comment:
I was thinking of splitting `PullRequestUpdater` into two classes, one for updating PRs and one for checking on them with a common abstract class like `PullRequestHandler`, but didn't want to do it while Adam has his refactoring PR up, two refactorings at the same time might be a bit much